presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -186,6 +186,49 @@ public void testSchemaOperations()
         assertUpdate(admin, "DROP SCHEMA new_schema");
     }
 
+    @Test
+    public void testSchemaAuthorization()
+    {
+        Session admin = Session.builder(getQueryRunner().getDefaultSession())
+                .setIdentity(Identity.forUser("hive")
+                        .withRole("hive", new SelectedRole(ROLE, Optional.of("admin")))
+                        .build())
+                .build();
+
+        assertUpdate(admin, "CREATE SCHEMA shared_schema");
```

Comment:
replace `shared_schema` with `test_schema_authorization`. So that name is unique across tests.